src/app/core/connectionTarget.ts

```diff
@@ -40,13 +43,13 @@ export default class ConnectionTarget {
      */
     toJSON() {
         // This is for the older design data
-        if (this.__component instanceof Component) {
+        if (this._component instanceof Component) {
             return {
-                component: this.__component.getID(),
-                port: this.__portLabel
+                component: this._component.id,
+                port: this._portLabel
             };
         } else {
-            return { component: this.__component, port: this.__portLabel };
+            return { component: this._component, port: this._portLabel };
         }
```

Comment:
We are no longer supporting older design data right? so only the first option should be available